packages/js/src/classic-editor.js

```diff
@@ -33,11 +36,25 @@ domReady( async () => {
 		initPrimaryCategory( jQuery );
 	}
 
-	const { SeoProvider } = await createSeoIntegration( {
+	const { SeoProvider, analysisWorker } = await createSeoIntegration( {
 		analysis: getAnalysisConfiguration(),
 		initialState: getInitialState(),
 	} );
 
+	/*
+	 * Register the analysis worker on `window.YoastSEO.analysis.worker`
+	 * and refreshing the analysis on `window.YoastSEO.app.refresh`
+	 * for backwards compatibility with the add-ons.
+	 */
+	registerApis( [
+		{ analysis: { worker: analysisWorker } },
+		{
+			app: {
+				refresh: debounce( () => dispatch( SEO_STORE_NAME ).analyze(), refreshDelay ),
```

Comment:
Very minor, but the analyze function is there after the SEO integration has been created too right, so you could do without the lambda in between.

```suggestion
				refresh: debounce( dispatch( SEO_STORE_NAME ).analyze, refreshDelay ),
```